build/dev/publish-devfile-registry-to-gh-pages.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+#
+# Copyright (c) 2018-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+set -e
+
+DEFAULT_BUILD_DIR="/projects/che-plugin-registry/output"
+BUILD_DIR=${BUILD_DIR:-$DEFAULT_BUILD_DIR}
+
+DEFAULT_GH_PAGES_DIR="/projects/gh-pages"
+GH_PAGES_DIR=${GH_PAGES_DIR:-$DEFAULT_GH_PAGES_DIR}
+
+# not setting eclipse-che in purpose to avoid errors committers pushing in the main registry
+DEFAULT_GITHUB_ACTOR="sunix"
```

Comment:
default should not be sunix

Reply:
I finally don't set any default value: the script will check and fail if GITHUB_ACTOR and GITHUB_TOKEN are not set